modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/CreateAnAPIApplicationEndpoint.testcase

```diff
@@ -94,19 +111,276 @@ definition {
 	}
 
 	@priority = 4
-	test PicklistScopeHasCorrectOptions {
+	test CanCreateEndpointWithSuccessAlert {
```

Comment:
Same here, please set the priority to what it is in the jira ticket, and remove portal.acceptance for priority 3 cases.